cmd/create/accountroles/validation.go

```diff
@@ -0,0 +1,36 @@
+package accountroles
+
+import errors "github.com/zgalor/weberr"
+
+func validateSharedVpcInputs(hostedCp bool, vpcEndpointRoleArn string,
+	route53RoleArn string) (bool, error) {
+
+	if !hostedCp {
+		if vpcEndpointRoleArn != "" {
+			return false, errors.UserErrorf("Can only use '%s' flag for Hosted Control Plane account roles",
```

Comment:
@hunterkepley I'd consider extracting these error messages to a `const` to avoid the duplication e.g:

```
return false, errors.UserErrorf(hostedCpOnlyErrorMessage, vpcEndpointRoleArnFlag)
```

where `hostedCpOnlyErrorMessage` is the string const.